src/network/__init__.py

```diff
@@ -4,19 +4,19 @@
 
 try:
     from .announcethread import AnnounceThread
-    from .connectionpool import BMConnectionPool
+    from .connectionpool import BMConnectionPool as BMConnectionPool_cls
+    BMConnectionPool = BMConnectionPool_cls()
```

Comment:
how about we use a different name for the object than the class name? How about `network.connectionpool` for instance and we keep `network.BMConnectionPool` as the class name?

Reply:
done, renamed as pool